easybuild/tools/github.py

```diff
@@ -888,6 +888,8 @@ def new_pr(paths, ecs, title=None, descr=None, commit_msg=None):
                 main_title = ', '.join(names_and_versions[:3] + ['...'])
 
             title = "{%s}[%s] %s" % (class_label, toolchain_label, main_title)
+        elif commit_msg:
```

Comment:
Are `file_info['ecs'] and all(file_info['new']) and not deleted_paths` and `commit_msg` mutually exclusive? I guess they aren't, but if they are, I'd suggest to swap the order of the ifs, since evaluating `file_info['ecs'] and all(file_info['new']) and not deleted_paths` is way more expensive than `commit_msg`. Nitpicking, I know.